src/cmd/cli/command/commands_test.go

```diff
@@ -141,6 +142,20 @@ func TestCommandGates(t *testing.T) {
 
 	t.Setenv("AWS_REGION", "us-west-2")
 
+	userinfoServer := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		if r.URL.Path != "/userinfo" {
+			t.Fatalf("unexpected path %q", r.URL.Path)
+		}
+		w.Header().Set("Content-Type", "application/json")
+		_, _ = w.Write([]byte(`{
+			"allTenants":[{"id":"default","name":"Default Workspace"}],
```

Comment:
this is technically not a conformant userinfo response. https://github.com/DefangLabs/portal/issues/273